monitoring/database.py

```diff
@@ -0,0 +1,20 @@
+from sqlalchemy import BigInteger, ForeignKey, Column, DateTime, Integer, String
+from sqlalchemy.ext.declarative import declarative_base
+
+
+MonitoringBase = declarative_base()
+
+
+class Build(MonitoringBase):
+    __tablename__ = 'builds'
+
+    id = Column(Integer, primary_key=True)
+    job_name = Column(String(4096), nullable=False)
+    build_id = Column(Integer, nullable=False)
+    build_url = Column(String(4096), nullable=False)
+    finished_at = Column(BigInteger, nullable=False)
+    status = Column(String(4096), nullable=False)
+    duration = Column(BigInteger, nullable=False)
```

Comment:
Let's include `secs` / `millis` in column names `finished_at` and `duration` as we've discussed yesterday.